src/improvements/template/TransferL2PAOFromL1.sol

```diff
@@ -0,0 +1,95 @@
+/// ATTENTION: Please use caution when using this template. Transferring ownership is high risk.
+///
+/// Post-Execution Checks
+/// 1. Find the L2 deposit transaction by identifying the alias of the L1 ProxyAdmin owner safe.
+/// 2. The transaction you're looking for should be the most recent transaction sent from the aliased L1PAO adress on L2. If it's not, then it should be a recent transaction from that was interacting with the L1 ProxyAdmin 0x4200000000000000000000000000000000000018.
+/// 3. Once you've found the correct transaction, verify that the expected log event was emitted i.e. 'emit OwnershipTransferred(oldOwner, newOwner)'.
+contract TransferL2PAOFromL1 is L2TaskBase {
+    using stdToml for string;
+
+    /// @notice The aliased L1 PAO owner.
+    address public aliasedNewOwner;
+
+    /// @notice Returns the safe address string identifier
+    function safeAddressString() public pure override returns (string memory) {
+        return "ProxyAdminOwner";
+    }
+
+    /// @notice Returns the storage write permissions required for this task.
+    function _taskStorageWrites() internal pure virtual override returns (string[] memory) {
+        string[] memory storageWrites = new string[](1);
+        storageWrites[0] = "OptimismPortalProxy";
+        return storageWrites;
+    }
+
+    /// @notice Sets up the template with the new owner from a TOML file.
+    function _templateSetup(string memory taskConfigFilePath) internal override {
+        super._templateSetup(taskConfigFilePath);
+        string memory toml = vm.readFile(taskConfigFilePath);
+
+        // New owner address. This address is unaliased.
+        address newOwnerToAlias = abi.decode(vm.parseToml(toml, ".newOwnerToAlias"), (address));
+        // Apply the alias to the new owner.
+        aliasedNewOwner = AddressAliasHelper.applyL1ToL2Alias(newOwnerToAlias);
+
+        // Only allow one chain to be modified at a time with this template.
+        SuperchainAddressRegistry.ChainInfo[] memory _chains =
+            abi.decode(vm.parseToml(toml, ".l2chains"), (SuperchainAddressRegistry.ChainInfo[]));
+        require(_chains.length == 1, "Must specify exactly one chain id to transfer ownership for");
+    }
+
+    /// @notice Builds the actions for transferring ownership of the proxy admin on the L2.
+    function _build() internal override {
+        uint64 gasLimit = 200000; // This gas limit was used for an example task previously: tasks/sep/010-op-l2-predeploy-upgrade-from-l1/input.json
+        SuperchainAddressRegistry.ChainInfo[] memory chains = superchainAddrRegistry.getChains();
+        OptimismPortal optimismPortal =
+            OptimismPortal(superchainAddrRegistry.getAddress("OptimismPortalProxy", chains[0].chainId));
+        optimismPortal.depositTransaction(
+            address(Predeploys.PROXY_ADMIN),
+            0,
+            gasLimit,
+            false,
+            abi.encodeWithSelector(ProxyAdmin.transferOwnership.selector, aliasedNewOwner)
```

Comment:
Let's use `abi.encodeCall` for type safety

Reply:
done https://github.com/ethereum-optimism/superchain-ops/pull/969/commits/8dc540942082539282624e72c2cc046f9f977601